packages/shorebird_cli/lib/src/commands/subscription/cancel_subscription_command.dart

```diff
@@ -34,7 +34,7 @@ class CancelSubscriptionCommand extends ShorebirdCommand
 
     final User user;
     try {
-      user = await client.getCurrentUser();
+      user = (await client.getCurrentUser())!;
```

Comment:
If you're already logged in we could cache the user and avoid having to make this extra request

Reply:
Not sure I follow – we won't be running login or signUp in this command, and we'll definitely want to get the latest subscription info for this user before dealing with subscription info.

Reply:
Oh sorry I think I misread this. We might want to remove the `auth.isAuthenticated` check here and instead handle the case where `getCurrentUser` is null rather than force unwrapping though 🤔

Reply:
I think this should still fail in the same way as before – if the user is null or if getCurrentUser fails in some other way, we log the error and exit. What's your concern?

Reply:
Ah! That makes sense. Updated.